app/views/plans/_edit_details.html.erb

```diff
@@ -1,132 +1,208 @@
-	        <%= f.email_field :principal_investigator_email, class: 'input-small left-indent' %>
-	        <span role="" id="principal_investigator_email_error" class="error-tooltip left-indent"></span>
-	      </div>
-
-	      <br />
-	      <div>
-	        <%= f.label(:data_contact, raw("<strong>#{_('Data Contact Person')}</strong>")) %>
-	      </div>
-	      <div>
-	        <input type="checkbox" id="show-data-contact" class="left-indent"<%= (@plan.data_contact.blank? && @plan.data_contact_email.blank? ? ' checked="checked"' : '') %> />
-	        <label class="inline checkbox-label" for="show-data-contact">
-	          <%= _('Same as Principal Investigator') %>
-	        </label>
-	      </div>
-	      <div class="form-input data-contact-info">
-	        <%= f.label _('Name'), for: :data_contact, 'aria-label': _('Data Contact Name') %>
-	        <%= f.text_field :data_contact, class: 'input-small left-indent' %>
-	      </div>
-	      <div class="form-input data-contact-info">
-	        <%= f.label _('Phone'), for: :data_contact_phone, 'aria-label': _('Data Contact Phone') %>
-	        <%= f.telephone_field :data_contact_phone, class: 'input-small left-indent' %>
-	      </div>
-	      <div class="form-input data-contact-info">
-	        <%= f.label _('Email'), for: :data_contact_email, 'aria-label': _('Data Contact Email') %>
-	        <%= f.email_field :data_contact_email, class: 'input-small left-indent' %>
-	        <span role="" id="data_contact_email_error" class="error-tooltip left-indent"></span>
-	      </div>
-      
-	      <br />
-	      <br />
-	      <div class="form-input">
-	        <%= render partial: 'shared/accessible_submit_button',
-	                   locals: {id: 'save-details-button', 
-	                            val: _('Save'), 
-	                            disabled_initially: true,
-	                            classes: 'small-input-button left-indent',
-	                            tooltip: _('You must provide a project title and any email addresses must be valid.')} %>
-	      </div>
-	    </fieldset>
-		</div>
-
-		<div class="col-md-4">
-	    <div class="bordered left-indent">
-	      <h2><%= _('Plan Guidance Configuration') %></h2>
-	      <p><%= _('To help you write your plan, %{application_name} can show you guidance from a variety of organisations.') % {application_name: Rails.configuration.branding[:application][:name]} %></p>
-	      <p><%= _('Select up to 6 organisations to see their guidance.') %></p>
-
-	      <ul id="priority-guidance-orgs">
-	        <%= render partial: "guidance_choices", 
-	                   locals: {choices: @important_ggs, form: f,
-	                            current_selections: @selected_guidance_groups} %>
-	      </ul>
+<div class="row">
+    <div class="col-md-8">
+    <%= form_for plan, html: {method: :put, class: 'form-horizontal edit_plan' } do |f| %>
+      <div class="form-group"><!-- title -->
+        <div class="col-md-12">
```

Comment:
You shouldn't nest columns like this. Columns should appear as children of `.row`. You should structure it like this:
```html
<div class="row">
  <div class="col-md-12>
     [title content]
  </div>
</div>
<div class="row">
  <div class="col-md-8">
    <div class="row">
      <div class="col-md-12">
        [form content]
      </div>
    </div>
  </div>
  <div class="col-md-4">
    [guidance section]
  </div>
</div>
```
or this if you want to use the grid to align the tooltips you can nest rows within a column (https://getbootstrap.com/docs/3.3/css/#grid-nesting):
```html
<div class="row">
  <div class="col-md-8>

    <div class="row">
      <div class="col-md-12">
        [title content]
      </div>
    </div>

    <div class="row">
      <div class="col-md-8">
        [1st form input]
      </div>
      <div class="col-md-4">
        [1st tooltip/popover]
      </div>
    </div>

    <div class="row">
      <div class="col-md-12">
        [2nd form input - has no tooltip/popover]
      </div>
     </div>

     ... repeat for each form input ... 
  </div>

  <div class="col-md-4">
    [guidance section]
  </div>
</div>
```

Reply:
I'm using horizontal form, i.e. form-horizontal from bootstrap. This means that .form-group class behaves as a grid row, so there is no need for adding .row. See documentation: https://getbootstrap.com/docs/3.3/css/#forms-horizontal

Note, that I respect row and column grid structure to separate the left container (plan details) from the right (guidance choices).

Reply:
ugh. One of Bootstrap's shortcut anti-patterns. `.form-group` == `.row` when inside a `.form-horizontal` and you use the same col classes.